src/api/map/routes/map.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable require-await */
```

Comment:
Use per-line statement and not per-file. Move them to immediately above the lines where lint rules are violated.